db/index.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
     password: Config.DB_PASSWORD,
     port: Config.DB_PORT,
     user: Config.DB_USER
-  }
+  },
+  debug: true
```

Comment:
Perhaps this value should depend on a flag from `Config`. In `production`, we probably wouldn't actually want debug mode enabled.